hotel-reservation-system/src/components/Hotel/ServiceTable.js

```diff
@@ -60,26 +63,34 @@ export default function ServiceTable({ hotelId }) {
       callUpdateAlert={callUpdateAlert}
     ></AddServiceForm>
   );
+  useEffect(() => {
+    if (hotelId === undefined) {
+      setServices(serviceList);
+    }
+  }, [serviceList, service]);
 
   useEffect(() => {
-    const loadServices = async () => {
-      await API.get(
-        "/services/" +
-          hotelId +
-          "/pages?PageNumber=" +
-          pageForRequest +
-          "&PageSize=" +
-          rowsPerPage
-      )
-        .then((response) => response.data)
-        .then((data) => {
-          console.log(data);
-          setServices(data.item1);
-          setMaxNumberOfServices(data.item2);
-        })
-        .catch((error) => console.log(error.response.data.message));
-    };
-    loadServices();
+    if (hotelId !== undefined) {
```

Comment:
You can use just "!!hotelId"

Reply:
all places with item!==undefined replaced by !!item